src/drivers/pci.rs

```diff
@@ -336,6 +350,7 @@ pub(crate) fn print_information() {
 
 #[allow(clippy::large_enum_variant)]
 #[allow(clippy::enum_variant_names)]
+#[non_exhaustive]
```

Comment:
Is this related to the other changes or a separate change?

Reply:
This is related because it fixes a ton of the warnings when matching on the variants with an unreachable `_` arm. I figured this would be a better solution to that instead of allowing a clippy lint in every method that matches on the enum